driverconfig/config.yaml

```diff
@@ -116,6 +116,7 @@ drivers:
       - configVersion: v2.6.0
         useDefaults: true
         supportedVersions:
+          - version: v121
```

Comment:
Will the old sidecars be used or the latest ones with K8s 1.21

Reply:
Latest side cars will be used with K8s 1.21